tools/serverpod_cli/lib/src/test_util/builders/generator_config_builder.dart

```diff
@@ -42,7 +43,8 @@ class GeneratorConfigBuilder {
           ),
         ],
         _extraClasses = [],
-        _enabledFeatures = [ServerpodFeature.database];
+        _enabledFeatures = [ServerpodFeature.database],
+        _relativeServerTestToolsPathParts = ['integration_test', 'test_tools'];
```

Comment:
Suggestion: Remove default value here.

By default this is not generated, so this field should not be configured.
Should also have a test that validates nothing is generated when this is not set in the configuration.

Reply:
Good point, will add that test! 🧪